samples/scenarios/templates/db_impersonation/connection-fields.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<connection-fields>
+<field name="server" label="Server" value-type="string" category="endpoint" />
+
+  <field name="port" label="Port" value-type="string" category="endpoint" default-value="21050" />
```

Comment:
```suggestion
<field name="port" label="Port" value-type="string" category="endpoint" default-value="21050" />
```
whitespace

Reply:
Thanks. I have now fixed indentation for the entire file.